main.js

```diff
@@ -1,47 +1,54 @@
+/* eslint-disable class-methods-use-this */
 /* eslint-disable no-unused-vars */
 const showBooks = document.querySelector('.record');
 const Title = document.querySelector('.title');
 const Author = document.querySelector('.author');
 const btn = document.querySelector('.form button');
 
-let books = [];
+class Book {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(obj) {
+    this.books = JSON.parse(localStorage.getItem('data'));
+    this.books.push(obj);
+    localStorage.setItem('data', JSON.stringify(this.books));
+    window.location.reload();
+  }
+
+  removeBook(title) {
+    this.books = JSON.parse(localStorage.getItem('data'));
+    this.books = this.books.filter((i) => i?.title !== title);
+    localStorage.setItem('data', JSON.stringify(this.books));
+    window.location.reload();
+  }
+}
+
+const bookc = new Book();
 
 btn.addEventListener('click', () => {
-  const book = {
+  const bookObj = {
     title: Title.value,
     author: Author.value,
   };
-  books = (JSON.parse(localStorage.getItem('data')));
-  books.push(book);
-  localStorage.setItem('data', JSON.stringify(books));
-  showBooks.innerHTML = books.map((i) => `<h5>${i?.title}</h5>
-      <h5>${i?.author}</h5>
-      <button onClick='removeBook(${JSON.stringify(i?.title)})'>Remove</button>
-      <hr>`)
-    .join('');
+  bookc.addBook(bookObj);
```

Comment:
- [ ] when trying to remove a duplicate book it removes both books instead of 1 of them, you might want to fix this using an Id for each book.